src/openms/include/OpenMS/CHEMISTRY/ModificationsDB.h

```diff
@@ -80,6 +80,16 @@ namespace OpenMS
       return db_;
     }
 
+    inline static ModificationsDB * getCustomInstance(OpenMS::String unimod_file, OpenMS::String psimod_file)
```

Comment:
Exactely, in some cases being able to modify the databases would be very helpful. I could also implement functionality to filter this based on the standard DBs, but this would be much more extensive to do than such a workaround. For my implementation, I would instantiate a custom ModificationsDB once and not change it afterwards within the whole TOPP tool.